Rethinking_2/Chp_06.ipynb

```diff
@@ -0,0 +1,3109 @@
+{
```

Comment:
<ul><li>This is an annoying <code>FutureWarning</code> that will disappear if/when we do a release of Theano. In the meantime, you can disable by adding this at the top of the NB:</li></ul><pre class="ql-syntax" spellcheck="false">import warnings
warnings.simplefilter(action="ignore", category=FutureWarning)
</pre><p> </p><ul><li>If you're running the master branch of PyMC, <code>pm.sample</code> now defaults to 1000 samples -- otherwise, just leave it there.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>